src/Css/Style.php

```diff
@@ -510,7 +512,7 @@ public function __construct(Stylesheet $stylesheet, $origin = Stylesheet::ORIG_A
             $d["elevation"] = "level";
             $d["empty_cells"] = "show";
             $d["float"] = "none";
-            $d["font_family"] = $stylesheet->get_dompdf()->getOptions()->getDefaultFont();
+            $d["font_family"] = empty($defaultFont) ? "serif" : $defaultFont;
```

Comment:
Given that we had a bug due to using `empty` for checking for an empty string not too long ago (see #2566), I think I would avoid using the function here, even if a font with name `0` is highly unlikely in practice. Simplest approach is probably using `$defaultFont === ""` here and making sure that `$options->getDefaultFont()` always returns a string.